app/models/helpers/health_check_types.rb

```diff
@@ -4,5 +4,14 @@ class HealthCheckTypes
     PROCESS = 'process'.freeze
     HTTP = 'http'.freeze
     NONE = 'none'.freeze
+
+    def self.constants_to_array
```

Comment:
maybe `all_types` would be a clearer name? You could also have a helper method `readiness_types` that does the subtraction in the ReadinessHealthCheckPolicy

https://github.com/cloudfoundry/cloud_controller_ng/pull/3361/files#diff-b390b8806efca835a9f8d37a91c6cf6d9d0aef019708a78d6c5086272b18d3abR4

Reply:
✅ done!